### DISCUSSION TITLE
Can't get data if use tag 'render'

### DISCUSSION THREAD

**Discussion by shkarinn** [harttle/liquidjs#717] [Q&A]
Hello. If I use old tag include
`{% include 'header/section' %}`
my data inside include file is working

For example I can get all data from 11ty global inside _header/section.liquid_ file
`{{ collections.all[0].template.templateData.globalData | debug }}`

But if use new tag 
`{% render 'header/section' %}`
I got undefined `<pre>undefined</pre>`

I can get access to data only outside file _section.liquid_
`{% render 'header/section' for contact.buttons as item %}`

liquid have section tag, but js version not.
liquidjs have only _render, include, layout_ tags. 
But include is Deprecated and working good.
Render tag have problem.

**Comment by harttle** [comment]
You have to set it to globals for access in `render`.  That's intended and thus different from `include`.
I guess maybe we shouldn't deprecate `include` as we don't have plans to support `section`.

**Comment by shkarinn** [comment]
> You have to set it to globals for access in `render`. That's intended and thus different from `include`. I guess maybe we shouldn't deprecate `include` as we don't have plans to support `section`.

I don't yet know how to set access to a global object for a single tag....
So far I have found the following solution. I will continue to use include for sections and ready blocks.
And render for mixins (macros).
